ios/KlaviyoBridge.swift

```diff
@@ -51,8 +51,18 @@ public class KlaviyoBridge: NSObject {
 
     @MainActor
     @objc
-    public static func registerForInAppForms() {
-        KlaviyoSDK().registerForInAppForms()
+    public static func registerForInAppForms(configuration: [String: AnyObject]? = nil) {
+        guard let configurationLength = configuration?["sessionTimeoutDuration"] as? TimeInterval else {
+            KlaviyoSDK().registerForInAppForms()
+            return
+        }
+        KlaviyoSDK().registerForInAppForms(configuration: IAFConfiguration(sessionTimeoutDuration: configurationLength))
```

Comment:
```suggestion
        if let configurationLength = configuration?["sessionTimeoutDuration"] as? TimeInterval {
            KlaviyoSDK().registerForInAppForms(configuration: IAFConfiguration(sessionTimeoutDuration: configurationLength))
        } else {
            KlaviyoSDK().registerForInAppForms()
        }
```
The end result here is the same, but for clarity I'd prefer an `if let` here instead of a `guard let`, for a similar reason as I describe [here](https://github.com/klaviyo/klaviyo-swift-sdk/pull/346#discussion_r2116512796). A `guard let` suggests that a condition is required in order for us to continue, and the early return is a fallback in case we don't meet that condition. In other words words, the "preferred" state is for the `guard let` to succeed, and the early return is just a safe way to handle the not-preferred scenario.

In this case, it's perfectly normal and expected to not have a `configurationLength` value, and we don't really have a (strong) preference here as to whether the `configurationLength` is provided or not, so I think `if let` is more appropriate here